lib/tabulo/table.rb

```diff
@@ -207,11 +207,10 @@ def initialize(sources, *columns, align_body: :auto, align_header: :center, alig
     #   by the Table-level setting passed to the <tt>align_header</tt> (which itself defaults
     #   to <tt>:center</tt>). Otherwise, this option determines the alignment of the header
     #   content for this column.
-    # @param [Symbol, String, Integer, nil] before The label of the column before (i.e. to
-    #   the left of) which the new column should inserted. If <tt>nil</tt> is passed, it will be
-    #   inserted after all other columns. If there is no column with the given label, then an
-    #   {InvalidColumnLabelError} will be raised. A non-Integer labelled column can be identified
-    #   in either String or Symbol form for this purpose.
+    # @param [Symbol, nil] before The label of the column before (i.e. to the left of) which the new
```

Comment:
i just changed the types here, i think this does not support String or Integer values (?)

Reply:
Good pickup. It does actually support an integer; column labels can be integers, strangely enough. You are correct about it not supporting a string, though.

Reply:
OK, I added Integer back in.